.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     '@typescript-eslint/no-empty-function': 'warn',
     '@typescript-eslint/no-var-requires': 'warn',
     '@typescript-eslint/ban-ts-comment': 'warn',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
Should this be under the `/* Temporary disabled rules for React development */` section? Just to emphasise that this is only while our TS conversion/coverage work is still WIP. 🤔

Reply:
I have a feeling that we'll disable this *forever* 😆 But yeah, makes sense

Reply:
We can always hope for now... haha